### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by teodutu**
Prepare the repo for content development.

### CODE REVIEW

**Review by gabrielmocanu — APPROVED** [review]
I don't see any issue. I think it's good to go.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security Summer School: Essentials Track
+Welcome to the Essentials Track of the Security Summer School.
+
+Each session is located in its specific folder. There, you will find a
+`README.md` file with the session documentation and, if it's the case,
+subfolders with support data for the challenges.
+
+Sessions are:
+- TODO
```
Comment (razvand) [inline README.md:9]: Why not list sessions?

I would enumerate them using the [ordered list syntax for GitHub Markdown](https://guides.github.com/features/mastering-markdown/).

**Review by razvand — CHANGES_REQUESTED** [review]
I would merge the `Tutorials` and `Challenges` sections in a single `Activities` section. Because tutorials and challenges may be interleaved.

I suggest we use a blank line after every heading title. It makes it more clear to read / write Markdown.

File: README.md (hunk 1 above)
Comment (teodutu) [reply README.md:9]: I was leaving room for changes to the order of the sessions. But on second thought, the current ordering should not require modifications, so I'll just add it here.

**Review by razvand — APPROVED** [review]
I suggest we write one (entire) phrase per line to make nice diffs.

Other than that, it's all good on my side.

**Review by razvand — APPROVED** [review]
All good

**Pull request merged**